hawk/src/test/java/com/orhanobut/hawk/HawkTest.java

```diff
@@ -423,4 +425,69 @@ public void init() {
     assertThat(latch.await(LATCH_TIMEOUT_IN_SECONDS, TimeUnit.SECONDS)).isTrue();
   }
 
+  @Test public void statusNotInitialisedBeforeBuild() {
+    Hawk.init(context);
+    assertEquals(Hawk.getStatus(), Hawk.Status.NOT_INITIALISED);
```

Comment:
can we use assertJ statements here? It's more readable and everywhere else is kindly used that same approach overall in hawk.
```java
assertThat(Hawk.isInitialised()).isTrue(); 
```